src/psyclone/psyGen.py

```diff
@@ -2000,14 +2001,15 @@ def dag_name(self):
         return "ACC_data_" + str(self.abs_position)
 
     def gen_code(self, parent):
-        '''
-        Generate the elements of the f2pygen AST for this Node in the Schedule.
+        '''Generate the elements of the f2pygen AST for this Node in the Schedule.
 
         :param parent: node in the f2pygen AST to which to add node(s).
         :type parent: :py:class:`psyclone.f2pygen.BaseGen`
+
+        :raises GenerationError: if no data is found to copy in.
+
         '''
-        from psyclone.f2pygen import DeclGen, DirectiveGen, CommentGen, \
-            IfThenGen, AssignGen, CallGen, UseGen
+        from psyclone.f2pygen import DeclGen, DirectiveGen, CommentGen
```

Comment:
pylint complains that `DirectiveGen` has already been imported.

Reply:
Removed this and all other DIrectiveGen imports in this file that are not required.